frontend/palworld-pal-editor-webui/src/i18n/ja.js

```diff
@@ -44,8 +44,8 @@ export default {
     Editor_23: "SoulBonus 攻撃力: ",
```

Comment:
Btw, do you know how to translate the SoulBonus into Japanese?

Reply:
`"SoulBonus 攻撃力"` is simply translated as `"ソウルボーナス 攻撃力"` in Japanese.
However, as "ソウルボーナス" is not used in the game, I prefer to omit "ソウルボーナス" and add translation for the soul bonus header as follows:

```i18n/ja.js
export default {
    // ...
    Editor_21: "パル強化 (力の石像)",
    Editor_22: "HP: ",
    Editor_23: "攻撃力: ",
    Editor_24: "防御力: ",
    Editor_25: "クラフト速度: ",
    // ...
}
```

This can avoid too wide item label length and I think this is natural for Japanese users.
Shall I modify them?
If possible, I'd like to change translation about "個体値" and the computed status.

Reply:
yes, please!